src/sentry/api/serializers/models/event.py

```diff
@@ -221,3 +221,44 @@ def serialize(self, obj, attrs, user):
         result['entries'] = [e for e in result['entries']
                              if e['type'] != 'breadcrumbs']
         return result
+
+
+class SnubaEvent(object):
+    """
+        A simple wrapper class on a row (dict) returned from snuba representing
+        an event. Provides a class name to register a serializer against, and
+        Makes keys accessible as attributes.
+    """
+
+    # The list of columns that we should request from snuba to be able to fill
+    # out a proper event object.
+    selected_columns = [
+        'event_id',
+        'message',
+        'user_id',
+        'username',
+        'email',
+    ]
+
+    def __init__(self, kv):
+        self.__dict__ = kv
```

Comment:
Might not be a bad idea to assert that `set(kv.keys()) == set(selected_columns)` here.

Reply:
done